app/views/benefits/reimbursed/OtherBenefitsView.scala.html

```diff
@@ -38,13 +40,14 @@
 
     @errorSummary(form.errors.distinct)
 
+    @heading(headingForUse, Some(messages("benefits.caption", (taxYear - 1).toString, taxYear.toString)), "govuk-!-margin-bottom-4")
```

Comment:
I think govuk-!-margin-bottom-4 needs to be changed to govuk-!-margin-bottom-2 to be the same as the other places. You have done it as it was the original, so it might be correct. Please check with Ben on Laura.